Mathlib/CategoryTheory/Limits/Shapes/RegularMono.lean

```diff
@@ -57,6 +59,52 @@ attribute [reassoc] RegularMono.w
 instance (priority := 100) RegularMono.mono (f : X ⟶ Y) [RegularMono f] : Mono f :=
   mono_of_isLimit_fork RegularMono.isLimit
 
+/-- Every isomorphism is a regular monomorphism. -/
+instance (priority := 100) RegularMono.ofIso (f : X ⟶ Y) [IsIso f] : RegularMono f where
```

Comment:
As this typeclass contains data, this should be a `def` rather than an instance.